modules/gdnative/nativescript/nativescript.h

```diff
@@ -38,8 +38,8 @@
 #include "script_language.h"
 #include "self_list.h"
 
-#include "godot_nativescript.h"
 #include "modules/gdnative/gdnative.h"
+#include <nativescript/godot_nativescript.h>
```

Comment:
Use `""` for Godot-specific includes, this is not a system wide one.

Reply:
It is. It's in the `CPPPATH`

Reply:
Same comment for all other `#include <.*>` you're using.

Reply:
To explain a little bit further:

The GDNative module is now split into an implementation part and an interface part. The interface part is all inside `modules/gdnative/include`. This makes it easier to extract those files so libraries can use them.

That's why I added that path into the include path for that module.

Reply:
Still, our convention is to use `""` for local headers, `<>` for system ones. Being in the CPPPATH doesn't make local files system headers :)

We don't use `#include <variant.h>` anywhere in Godot, even though `core` is in the CPPPATH.

Reply:
@akien-mga at least all files inside of `modules/gdnative/include` should use it, as those are the files that are later used in the creation of libraries, where they are used *like* a system path.

Reply:
Alright, you know best about the practical use case, I just wanted to point out the convention. Actual feature beats code style of course :)